Releasenotes.md

```diff
@@ -1,5 +1,15 @@
 # Releasenotes Haal-Centraal-BRP-Bevragen
 
+## Versie 1.1.0
+
+- In de resource IngeschrevenPersoon is in de naam de property "regelVoorafgaandAanAanschrijfwijze" toegevoegd
```

Comment:
hier moet dan ook adelijkeTitelPredikaat worden toegevoegd in de readme

Reply:
releasenotes aangepast, bestandsnaam gewijzigd --> kleine r + verplaatst naar /docs